app/routes/__frontend/newsletter/issues/2024-04.jsx

```diff
@@ -0,0 +1,370 @@
+				<p className="mb-0">
+					"So grateful to find so many encouraging people!"
+				</p>
+				<footer className="blockquote-footer">Alex</footer>
+			</blockquote>
+			<blockquote className="blockquote">
+				<p className="mb-0">"Grateful also as always for this community."</p>
+				<footer className="blockquote-footer">David</footer>
+			</blockquote>
+			<blockquote className="blockquote">
+				<p className="mb-0">
+					"I am also grateful for all the encouragement Julia gave me this week;
+					it helped me break out of my shell."
+				</p>
+				<footer className="blockquote-footer">Ethan</footer>
+			</blockquote>
+			<blockquote className="blockquote">
+				<p className="mb-0">
+					"Grateful for the VC Community, Good health, Family, getting assigned
+					another client this month."
+				</p>
+				<footer className="blockquote-footer">Brendan</footer>
+			</blockquote>
+
+			<hr />
+
+			<h2 className="my-5">📆 What's happening at Virtual Coffee</h2>
+			<h3>March Recap</h3>
+			<p>
+				<strong>💡Monthly Theme & Challenge: Getting Job Ready!</strong>
+			</p>
+			<p>
+				We spent the month working on resumes, cover letters, interview skills
+				and all thing getting ready to work! Shoutout to all our community
+				members who participated!
+			</p>
+
+			<h3 className="mt-4">April: Find Your Voice with Public Speaking!</h3>
+			<p>
+				<strong>💡Monthly Theme & Challenge: Public Speaking!</strong>
+			</p>
+			<p>
+				Whether you're preparing for a keynote, a tech talk, or just want to get
+				more comfortable speaking in public, we've got your back.
+			</p>
+			<p>
+				During this month, we'll be breaking down the art of public speaking
+				into manageable steps, focusing on different aspects each week.
+				Throughout the month, we'll provide resources, tips, and opportunities
+				to practice and get feedback from the community.
+			</p>
+			<p>
+				To view all of the details of this year's challenge, check out the{' '}
+				<Link to="/monthlychallenges/apr-2024">April 2024 challenge page</Link>{' '}
+				on the Virtual Coffee website.
+			</p>
+
+			<h3 className="mt-4">☕🪑 Coffee Table Events</h3>
+			<ul>
+				<li>Tech Interview Study Group (Mondays at 4:00 PM ET)</li>
+				<li>
+					Accountabilibuddies (Tuesdays at 7:00 PM ET and Thursdays at 9:00 AM
+					ET)
+				</li>
+				<li>The Pack Hunt (Tandem Job Hunting) (Wednesdays at 2:00 PM ET)</li>
+				<li>Feelings Friday (Fridays at 10:00 AM ET)</li>
+				<li>
+					Frontend Friday Folks fighting CSSBattle.dev (Fridays at 11:00 AM ET)
+				</li>
+			</ul>
+
+			<h3 className="mt-4">📅 Weekly Async Events</h3>
+			<ul>
+				<li>
+					Mondays: Goal-setting post in <code>#goals-and-wins</code>
+				</li>
+				<li>
+					Tuesdays: Trivia in <code>#game-night</code>
+				</li>
+				<li>
+					Wednesdays: Midweek check-in in <code>#general</code>
+				</li>
+				<li>
+					Thursdays: New book club questions for Cracking the Coding Interview
+					on GitHub
```

Comment:
**issue:** This has fallen off. I've spoken with Lauren and I know she's got a lot going on so she may have decided to take a hiatus.